application/classes/infosec/login_service.py

```diff
@@ -111,6 +95,7 @@ def authenticate_user_credentials(self):
                 'SELECT password FROM users WHERE username = ?', (self.__username,))
             if result:
                 stored_password = result[0][0]
+                print('db password', stored_password)
                 # Compare entered password with stored password
                 if bcrypt.checkpw(self.__password.encode('utf-8'), stored_password):
```

Comment:
The problem might have been that self.__password is being directly accessed, without the getter, so its encrypted by the encryption service still, and fails this check by bcyrpt

Reply:
You're probably right, that makes sense!

Reply:
I wont have time to fix this until much later, but feel free to subbranch this branch and have a play, otherwise ill do that tonight

Reply:
I'll have a look once I'm home!